src/client/js/controllers.js

```diff
@@ -88,43 +88,64 @@ netStatsApp.controller('StatsCtrl', function (
     $localStorage.pinned = $scope.pinned;
   };
 
-  setInterval(function () {
-    $scope.$apply();
-  }, 750);
-
   $scope.getNumber = function (num) {
     return new Array(num);
   };
 
   // Socket listeners
   // ----------------
   socket
-    .on('connect', function open () {
+    .on('connect', function () {
       socket.emit('ready');
       console.log('The connection has been opened.');
     })
-    .on('end', function end () {
+    .on('end', function () {
       console.log('Socket connection ended.');
     })
-    .on('error', function error (err) {
+    .on('error', function (err) {
       console.log(err);
     })
-    .on('reconnecting', function reconnecting (opts) {
+    .on('reconnecting', function (opts) {
```

Comment:
why get rid of the function names?

Reply:
Several reasons.

- The other methods did not have a name as well
- It's easier to read
- Those methods are anonymous anyway so it makes not difference